test_crates/minimum_rustc/fails/rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.64"
```

Comment:
Unfortunately I think the way our tests are set up will explicitly ignore this.

It might be less work to write this test as a GitHub Action job. That way we don't have to change how the test setup works, and people don't have to keep Rust 1.64 installed just to run tests locally.

Reply:
So, I managed to have the test actually use the `1.64` channel automatically. However I agree that forcing people to have Rust 1.64 installed is pretty annoying.
My original idea was that it would catch bumps of the minimal version, so that the error detection + message might be updated. But looking at it now, I guess such a bump would fail CI anyway, and it would make it clear that the check `rust_version < rustc_version_1_65` must be updated... I think ?

So, should I remove the tests entirely ?

Reply:
I would remove the tests here but then set up an equivalent test in GitHub Actions, where it explicitly uses Rust 1.64 to semver-check a crate, captures the output, and asserts the exit code was non-zero and contained the warning about using an old Rust.

There are some tests like that in GitHub Actions already, and you should be able to do what they do as well.